modules/frontend/server.py

```diff
@@ -8,32 +8,28 @@
 def index():
     return render_template('home.html')
 
-@app.route("/png-extraction", methods = ['GET'])
-def PNG_Extraction():
-    return render_template('pngHome.html')
-
-config_values = {}
 
-@app.route('/', methods=['POST'])
+@app.route('/png-extraction', methods=['POST', 'GET'])
 def extract_png():
+    config_values = {}
```

Comment:
Please avoid changing the frontend module (it is an experimental module as of now) as it is currently actively developed, most likely your changes will be overwritten.

Please focus on the png-extraction, cold-extraction, and meta-extraction modules as they are the mature/stable modules.